packages/Color/src/Color.tsx

```diff
@@ -12,6 +12,8 @@ export interface CustomCSSProps extends React.CSSProperties {
 }
 
 export interface ColorProps {
+    /** The appearance of the color */
```

Comment:
Can we add a small comment saying this props is only usable in the brand workleap